### ISSUE TITLE
Audio/Video Desync with Internal Player

### ISSUE THREAD

**Issue by NightHammer1000** [recloudstream/cloudstream#516]
### Steps to reproduce

Start Video
Audio gets Desynced after a few Minutes and is almost 2 sec off by the end

### Expected behavior

Audio should stay syncron

### Actual behavior

Audio gets Desynced after a few Minutes and is almost 2 sec off by the end.

Issue does not exist with VLC or MP and happens on every Source

Tested on FireTV 4K Max and S21 Ultra

### Cloudstream version and commit hash

4.0.1 446f774

### Android version

13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to pre-release version **[Latest](https://github.com/recloudstream/cloudstream/releases)**.
- [X] If related to a provider, I have checked the site and it works, but not the app.
- [X] I will fill out all of the requested information in this form.

**Comment by LagradOst** [comment]
unable to replicate, this sounds like an exoplayer issue not cs3 issue

**Comment by NightHammer1000** [comment]
It only startet with the latest build.
The one before was fine.
Also tested on an Redmi Note and an S10+ with diffrent Android Versions from 11 to 13. Also with an Regular Amazon Fire TV.
Same Issue on all.

Going back one Build and everything is fine again.

**Comment by LagradOst** [comment]
we have not touched the player in a while, are you 10000% sure it happens on every video and on every source and that the old ver is not doing the same

**Comment by NightHammer1000** [comment]
Yeah. Been testing the whole Sunday because I felt like going crazy.
Especially because its not happening on MPV and VLC as Players.
Only with the Internal. This Automatically rules the Source out.
